cfpb/consumerfinance.gov
# Remove accessible-languages macro

The accessible languages macro added a `lang` and `dir` attribute to RTL blocks for arabic pages. However, arabic pages now have `lang='ar'` set on the root `html` element and `dir='rtl'` set on the `main` element, so having this set on internal elements is unneeded. Additionally, this was used on the paying for college pages, which don't appear in arabic.

## Removals

- Remove accessible-languages macro

## How to test this PR

1. RTL pages should be unchanged, see https://github.com/cfpb/consumerfinance.gov/pull/7811 for list.
